### PR TITLE
Feature/update nlp front

### PR DESCRIPTION
**Pull request by LyZeoN**

### CODE REVIEW

File: assets/src/app/models/nlp_entity.ts [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+    label: string
+    score: string
+    category_id: string
 }
+
+export class Topic {
+    label: string
+    score: string
+    wiki_id: string
+}
```
Comment (rfibigr) [inline assets/src/app/models/nlp_entity.ts]: new line

File: assets/src/app/order/entity/entity.component.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  @Input() entities: Entity[];
+  @Input() word: string;
+  @Input() index: number;
+
+  constructor(
+  ) { }
+
+  ngOnInit() {
+  }
```
Comment (MarcAntoine-Arnaud) [inline assets/src/app/order/entity/entity.component.ts]: if empty, remove them ...

File: assets/src/app/order/entity/entity.component.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  getStyle(index: number) {
```
Comment (MarcAntoine-Arnaud) [inline assets/src/app/order/entity/entity.component.ts]: Is it possible to use less file to store entire style.
You can use class for that.

File: assets/src/app/order/nlp_viewer.component.ts [hunk 4]
```diff
@@ -75,34 +78,22 @@ export class NlpViewerComponent {
...
   }
 
-  getListOfWord(text: string[]) {
-    var entities = this.nlp.entity;
-    var len_entities = entities.length;
-    var i_entity = 0;
-    var words = this.createListOfWords();
+  mergedLeft(word: string): boolean {
+    return [',', '.', ';'].includes(word) || word.startsWith('-');
+  }
```
Comment (MarcAntoine-Arnaud) [inline assets/src/app/order/nlp_viewer.component.ts]: could you merge from entity component to remove duplicated code ?

File: assets/src/app/order/entity/entity.component.ts (hunk 3 above)
Comment (LyZeoN) [reply assets/src/app/order/entity/entity.component.ts]: i need variable in my style

**Comment by MarcAntoine-Arnaud** [comment]
Merge has append with open comments ....

**Pull request merged**